STL_Extension/include/CGAL/Compact_container.h

```diff
@@ -908,6 +908,14 @@ namespace internal {
       m_ptr.p = nullptr;
     }
 
+    CC_iterator (pointer p)
+#ifdef CGAL_COMPACT_CONTAINER_DEBUG_TIME_STAMP
+      : ts(0)
+#endif
+    {
+      m_ptr.p = p;
+    }
```

Comment:
When I saw that new constructor, I thought about new ambiguities. And I was right, Travis shows a lot of errors:
https://travis-ci.com/github/CGAL/cgal/builds/178369312